### DISCUSSION TITLE
Cannot see pending notifications

### DISCUSSION THREAD

**Discussion by antoniovazquezblanco** [community/community#21977] [Projects and Issues]
<p>According to Github I have pending notifications but when I access the notifications page I am unable to see them!</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/a/9aad582b1aa6e43895a6c0481da449f8db5fb507.png" data-download-href="/uploads/short-url/m4kUZ8nE7eMxwdjaZGBgXuuYAir.png?dl=1" title="imagen" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/304193/181067986-ea75b440-f93e-4fae-92d2-6157db9b383f.png" alt="imagen" data-base62-sha1="m4kUZ8nE7eMxwdjaZGBgXuuYAir" width="690" height="215" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/a/9aad582b1aa6e43895a6c0481da449f8db5fb507_2_690x215.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/a/9aad582b1aa6e43895a6c0481da449f8db5fb507_2_1035x322.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/9/a/9aad582b1aa6e43895a6c0481da449f8db5fb507_2_1380x430.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/a/9aad582b1aa6e43895a6c0481da449f8db5fb507_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">imagen</span><span class="informations">1912×596 18.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>What am I doing wrong? :confused:</p>

**Comment by jfullstackdev** [comment]
<p>is this still present?</p>
<p>yesterday there was an update from GitHub, and others experienced the some problems</p>

**Comment by antoniovazquezblanco** [comment]
<p>Unfortunately it is still present. It has been happening to me for a while now…</p>
<p>The issue seems to only be related to that one aircrack-ng notification. Always the same repo but cannot see which issue is triggering the notification.</p>
<p>Thanks!</p>

**Comment by metalelf0** [comment]
<p>I’m seeing the same issue. “Notification 1 of 0”, and I can’t figure out which one is causing the issue. It’s kinda frustrating cause I can’t understand when I have actual notifications, as the indicator is always on.</p>

**Comment by antoniovazquezblanco** [comment]
<p>This is still happening to me. Has it been solved for anyone?</p>

**Comment by antoniovazquezblanco** [comment]
<p>Could someone point me to the correct place to report the issue? Thanks</p>

**Comment by jfullstackdev** [comment]
<p>I noticed that you have 1 notif under Repos</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181067994-e3d3e232-79d0-46e9-8be8-88b11a5208f4.png" alt="image" data-base62-sha1="6Cw9HZi1VGMw5hWeGADaNv96Cob" width="363" height="95"></p>
<p>try to click that</p>

**Comment by antoniovazquezblanco** [comment]
<p>The same as if not selected…</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/c/2cb11adb811c319dbd3be04e9e18126fa40da07f.png" data-download-href="/uploads/short-url/6nmsa2apFvuDM0MIYll1v49Tkrd.png?dl=1" title="imagen" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/304193/181067996-76bf70b0-95e6-4c6c-bbfc-a9ec6796378e.png" alt="imagen" data-base62-sha1="6nmsa2apFvuDM0MIYll1v49Tkrd" width="690" height="232" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/c/2cb11adb811c319dbd3be04e9e18126fa40da07f_2_690x232.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/c/2cb11adb811c319dbd3be04e9e18126fa40da07f_2_1035x348.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/2/c/2cb11adb811c319dbd3be04e9e18126fa40da07f_2_1380x464.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/c/2cb11adb811c319dbd3be04e9e18126fa40da07f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">imagen</span><span class="informations">1906×643 23.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thank you for the tip!</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>this is where you can have the feedback, it’s the Product Team</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github/feedback">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github/feedback" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181067997-133c5672-2dce-42c7-a447-0c4e17958a52.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/4/245132770c85013167707f7ac9038e2e5d856bdf_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/4/245132770c85013167707f7ac9038e2e5d856bdf_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/4/245132770c85013167707f7ac9038e2e5d856bdf.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/4/245132770c85013167707f7ac9038e2e5d856bdf_2_10x10.png"></div>

<h3><a href="https://github.com/github/feedback" target="_blank" rel="noopener">GitHub - github/feedback: Public feedback discussions for: GitHub for Mobile,...</a></h3>

  <p>Public feedback discussions for: GitHub for Mobile, GitHub Discussions, GitHub Codespaces, GitHub Sponsors, GitHub Issues and more! - GitHub - github/feedback: Public feedback discussions for: GitH...</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by antoniovazquezblanco** [comment]
<p>Opened a discussion!</p>
<p>Thanks a lot <a class="mention" href="/jdevstatic">@jdevstatic</a> !</p>

**Comment by antoniovazquezblanco** [comment]
<p>It seems github has solved the issue.</p>
